odxtools/compumethods/createanycompumethod.py

```diff
@@ -124,28 +124,13 @@ def create_any_compu_method_from_et(et_element: ElementTree.Element,
         compu_internal_to_phys = odxrequire(et_element.find("COMPU-INTERNAL-TO-PHYS"))
 
         internal_to_phys: List[CompuScale] = []
-        for scale in compu_internal_to_phys.iterfind("COMPU-SCALES/COMPU-SCALE"):
-            lower_limit = Limit.from_et(scale.find("LOWER-LIMIT"), internal_type=internal_type)
-            upper_limit = Limit.from_et(scale.find("UPPER-LIMIT"), internal_type=internal_type)
-
-            if (vt := scale.find("COMPU-INVERSE-VALUE/VT")) is not None:
-                compu_inverse_value = internal_type.from_string(odxrequire(vt.text))
-            elif (v := scale.find("COMPU-INVERSE-VALUE/V")) is not None:
-                compu_inverse_value = internal_type.from_string(odxrequire(v.text))
-            else:
-                compu_inverse_value = None
-
+        for scale_elem in compu_internal_to_phys.iterfind("COMPU-SCALES/COMPU-SCALE"):
             internal_to_phys.append(
-                CompuScale(
-                    short_label=(scale.findtext("SHORT-LABEL")
-                                 if scale.find("SHORT-LABEL") is not None else None),
-                    description=create_description_from_et(scale.find("DESC")),
-                    lower_limit=lower_limit,
-                    upper_limit=upper_limit,
-                    compu_inverse_value=compu_inverse_value,
-                    compu_const=odxrequire(scale.findtext("COMPU-CONST/VT")),
-                ))
+                CompuScale.from_et(
+                    scale_elem, doc_frags, internal_type=internal_type,
+                    physical_type=physical_type))
 
+        kwargs["physical_type"] = DataType.A_UNICODE2STRING
```

Comment:
A_UNICODE2STRING means 2 bytes per char, are you sure?

Reply:
not really. this is littered across the codebase whenever a string is to be represented, though. probably we should change this to `A_UTF8STRING`... (let's see if something blows up)

Reply:
okay, I checked: on the python side, `A_UNICODE2STRING` always maps to `str` (which, if I'm not mistaken, uses UTF8 internally), so there is no difference  declaring the physical type `A_UNICODE2STRING` compared to `A_UTF8STRING`. I think Katrin just picked `A_UNICODE2STRING` on a whim here, so let's stick with it for now...